### PR TITLE
[MS] Workspace history

### LINKED ISSUE #8663: Workspace History: GUI
See user stories in #6147.

### PR DESCRIPTION
**Pull request by Max-7**
Closes #8663 

Waiting for #8662 

- [X] Keep changes in the pull request as small as possible
- [X] Ensure the commit history is sanitized
- [X] Give a meaningful title to your PR
- [X] Describe your changes
- [X] Link any related issue in the description

### CODE REVIEW

File: client/src/locales/en-US.json
Comment (mmmarcos) [inline client/src/locales/en-US.json:1650]: I would mention that the current versions can still be retrieved. Something like:
```suggestion
        "restoreConfirmQuestion": "Are you sure to restore the selected file or folder? This will overwrite any current version but you can still retrieve it in the History. | Are you sure to restore the selected files or folders? This will overwrite any current version but you can still retrieve them in the History.",
```

File: client/src/locales/en-US.json
Comment (mmmarcos) [inline client/src/locales/en-US.json:1651]:
```suggestion
        "empty": "No files or folders for the selected date.",
```

File: client/src/locales/fr-FR.json
Comment (mmmarcos) [inline client/src/locales/fr-FR.json:588]:
```suggestion
            "restore": "Restauré {name}",
```

File: client/src/locales/fr-FR.json
Comment (mmmarcos) [inline client/src/locales/fr-FR.json:592]:
```suggestion
            "failedDetails": "Impossible de restaurer ce fichier ou dossier."
```

File: client/src/locales/fr-FR.json
Comment (mmmarcos) [inline client/src/locales/fr-FR.json:1649]:
```suggestion
        "restoreConfirmTitle": "Confirmer la restauration des fichiers?",
```

File: client/src/locales/fr-FR.json
Comment (mmmarcos) [inline client/src/locales/fr-FR.json:1650]:
```suggestion
        "restoreConfirmQuestion": "Êtes-vous sûr de vouloir restaurer le fichier ou le dossier sélectionné ? Cette opération écrasera la version actuelle, mais vous pourrez toujours la retrouver dans l'historique. | Êtes-vous sûr de vouloir restaurer les fichiers ou dossiers sélectionnés ? Cette opération écrasera toute version actuelle, mais vous pourrez toujours les récupérer dans l'historique.",
```

File: client/src/locales/fr-FR.json
Comment (mmmarcos) [inline client/src/locales/fr-FR.json:1651]:
```suggestion
        "empty": "Aucun fichier ou dossier à la date sélectionnée.",
```

File: client/src/locales/fr-FR.json
Comment (mmmarcos) [inline client/src/locales/fr-FR.json:1652]:
```suggestion
        "error": "Impossible de récupérer l'historique de l'espace de travail pour la date sélectionnée",
```

**Review by mmmarcos — APPROVED** [review]
I've performed some quick tests and it seems to be working OK. IMO this feature needs some UI/UX tweaking, but it is probably better to do that in another PR (also to take the time to use this feature).

Just some comments on the wording.

**I did not review any ts/vue file**, so better wait for other approvals before merge.

File: client/src/locales/fr-FR.json
Comment (Max-7) [reply client/src/locales/fr-FR.json:585]: Quoting from wikipedia 
> La [Commission générale de terminologie et de néologie](https://fr.wikipedia.org/wiki/Commission_g%C3%A9n%C3%A9rale_de_terminologie_et_de_n%C3%A9ologie) indique qu'il faut utiliser le mot « internet » comme un nom commun, c'est-à-dire sans majuscule

File: client/src/locales/fr-FR.json
Comment (Max-7) [reply client/src/locales/fr-FR.json:588]: This is the message used while the file is being restored, or while in the queue to be restored.

File: client/src/locales/en-US.json
Comment (mmmarcos) [reply client/src/locales/en-US.json:588]: It felt a little weird because this is displayed in the "Last uploaded" section and there is no "Import of {file}" for the other actions: 
image

Since there is a specific icon for restore, maybe it is best to remove the "Restoration of"?

Or, if you prefer, I can open a new issue to discuss how we can harmonize this dialog:
- Add a "restored" section?
- Rename "Last uploaded" in "Done" or "Completed"?
- There multiple words for the same thing : "background _tasks_", "last _uploaded_", _import_...

**Review by mmmarcos — COMMENTED** [review]
Please add a newsfragment for the changelog

File: client/src/locales/fr-FR.json
Comment (NicoTuxx) [inline client/src/locales/fr-FR.json:1649]:
```suggestion
        "restoreConfirmTitle": "Confirmer la restauration des fichiers ?",
```

**Review by NicoTuxx — CHANGES_REQUESTED** [review]

**Review by NicoTuxx — APPROVED** [review]

**Pull request merged**
